controllers/getters.go

```diff
@@ -0,0 +1,60 @@
+package controllers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+func (c controller) GetPokemon(w http.ResponseWriter, r *http.Request) {
```

Comment:
Remember to apply comments to your functions 

// GetPokemon do... resolve... return...
func (c controller) GetPokemon(w http.ResponseWriter, r *http.Request) {